packages/core-components/src/components/SupportButton/SupportButton.tsx

```diff
@@ -100,9 +100,8 @@ export const SupportButton = ({ title, children }: SupportButtonProps) => {
         color="primary"
         onClick={onClickHandler}
       >
-        <Box marginRight={1}>
-          <HelpIcon />
-        </Box>
+        <HelpIcon />
+        <Box mr={1} />
```

Comment:
Agreed. Looks like some kind of spacer hack like `<div />`

Reply:
Yes - hacky indeed. The reasoning was the following, which triggered me a bit:
| Before | After |
|:---:|:---:|
| <img width="327" alt="Screenshot 2021-07-16 at 10 47 21" src="https://user-images.githubusercontent.com/8904624/125940143-e8f23100-6b06-4de3-b906-bbb601b4ce60.png"> | <img width="330" alt="Screenshot 2021-07-16 at 10 47 34" src="https://user-images.githubusercontent.com/8904624/125940149-640bd665-44c4-4084-bea5-d8f4223d7262.png"> |

The icon aligned because of the wrapping. I now enforced a static height on the wrapper, which I think is ok here, as the `SupportButton` is a really specific component, not used for generic things.